CineElorrietaMVC/src/main/java/Vista/PanelRegistro.java

```diff
@@ -0,0 +1,183 @@
+
+		lblTextoPanel = new JLabel("PANEL REGISTRO:");
+		lblTextoPanel.setFont(new Font("Arial", Font.BOLD, 20));
+		lblTextoPanel.setBounds(191, 22, 187, 46);
+		add(lblTextoPanel);
+
+		JLabel lblNIF = new JLabel("NIF:");
+		lblNIF.setFont(new Font("Arial", Font.PLAIN, 17));
+		lblNIF.setBounds(33, 225, 48, 23);
+		add(lblNIF);
+
+		textNIF = new JTextField();
+		textNIF.setBounds(78, 228, 192, 20);
+		add(textNIF);
+		textNIF.setColumns(10);
+
+		btnVolver = new JButton("Volver");
+		btnVolver.setBounds(137, 330, 89, 23);
+		add(btnVolver);
+
+		btnRegistrar = new JButton("Registrar");
+		btnRegistrar.setBounds(33, 330, 89, 23);
+		add(btnRegistrar);
+
+		JLabel lblDNI = new JLabel("DNI:");
+		lblDNI.setFont(new Font("Arial", Font.PLAIN, 17));
+		lblDNI.setBounds(330, 79, 81, 30);
+		add(lblDNI);
+
+		JLabel lblContrasenna = new JLabel("Contrase\u00F1a:");
+		lblContrasenna.setFont(new Font("Arial", Font.PLAIN, 17));
+		lblContrasenna.setBounds(33, 259, 127, 23);
+		add(lblContrasenna);
+
+		JLabel lblNombre = new JLabel("Nombre:");
+		lblNombre.setFont(new Font("Arial", Font.PLAIN, 17));
+		lblNombre.setBounds(28, 82, 89, 24);
+		add(lblNombre);
+
+		JLabel lblApellido = new JLabel("Apellido:");
+		lblApellido.setFont(new Font("Arial", Font.PLAIN, 17));
+		lblApellido.setBounds(28, 117, 106, 20);
+		add(lblApellido);
+
+		textDNI = new JTextField();
+
+		textDNI.setBounds(371, 86, 141, 20);
+		add(textDNI);
+		textDNI.setColumns(10);
+
+		textNombre = new JTextField();
+
+		textNombre.setBounds(101, 86, 169, 20);
+		add(textNombre);
+		textNombre.setColumns(10);
+
+		textApellido = new JTextField();
+
+		textApellido.setBounds(101, 120, 169, 20);
+		add(textApellido);
+		textApellido.setColumns(10);
+
+		passwordContrasena = new JPasswordField();
+
+		passwordContrasena.setBounds(137, 262, 133, 20);
+		add(passwordContrasena);
+
+		initializeEvents();
+
+	}
+
+	private void initializeEvents() {
+		this.btnVolver.addActionListener(listenerBotonVolver(this.controlador));
+		this.btnRegistrar.addActionListener(listenerBotonRegistrar(this.controlador));
+	}
+
+	private ActionListener listenerBotonVolver(ControladorPanelRegistro controladorPanelRegistro) {
+		return new ActionListener() {
+			public void actionPerformed(ActionEvent arg0) {
+				System.out.println("Ejecutando evento Boton Volver");
+				controladorPanelRegistro.accionadoBottonVolverPanelLogin();
+			}
+		};
+	}
+
+	private ActionListener listenerBotonRegistrar(ControladorPanelRegistro controladorPanelRegistro) {
+		return new ActionListener() {
+			public void actionPerformed(ActionEvent arg0) {
+				System.out.println("Ejecutando evento Boton Registrar");
+
+				String nombre = textNombre.getText();
+				String dni = textDNI.getText();
+				String nif = textNIF.getText();
+				String apellido = textApellido.getText();
+				@SuppressWarnings("deprecation")
+				String password = passwordContrasena.getText();
+
+				if (controladorPanelRegistro.comprobarFormatoNombre(nombre)
```

Comment:
seria mas optimo si en una llamada al controlador hacéis la comprobación de que todo está correcto .

Reply:
Corregido.

Reply:
Corregido.